hasura.planx.uk/migrations/1709565833346_run_sql_migration/up.sql

```diff
@@ -0,0 +1,15 @@
+-- Create the role
+CREATE ROLE metabase_read_only;
+
+-- Grant usage on schema
+GRANT USAGE ON SCHEMA public TO metabase_read_only;
+
+-- Grant select permissions on public tables used by Metabase (useful for joins)
+GRANT SELECT ON public.flows TO metabase_read_only;
+GRANT SELECT ON public.published_flows TO metabase_read_only;
+GRANT SELECT ON public.teams TO metabase_read_only;
```

Comment:
This might actually be redundant - I think there might be enough information in the views? If it's simpler to remove this I'm all for it 👍

Reply:
I'll contact Ollie today, or manually check Metabase, to see if this is required 👍

Reply:
Removed - these aren't currently used. See https://opensystemslab.slack.com/archives/C5Q59R3HB/p1709720214019159 (OSL Slack)

Reply:
Update - in order to move this forward I've granted access to the tables `analytics` and `analytics_logs`. They are safe to read, and currently used by SQL questions. The aim would be that once we migrate to GUI questions, we can rely on summary tables and tidy this up.

@zz-hh-aa Indicated that `lowcal_sessions` is used, which we do not want, so I've intentionally left this out to force a transition to `submission_services_summary`.

If this breaking change it too harsh, we can toggle the PG username/password used in Metabase until this is resolved.

Reply:
I think I've removed `lowcal_sessions` as a source from all the questions now anyway, and am only using summary tables + `analytics` or `analytics_logs`, so don't think anything should break! All good on my end.

Reply:
Ah that's perfect @zz-hh-aa thanks very much!